modules/extension/app-schema/app-schema/src/test/java/org/geotools/appschema/filter/ArithmeticFilterTest.java

```diff
@@ -0,0 +1,138 @@
+ *    modify it under the terms of the GNU Lesser General Public
+ *    License as published by the Free Software Foundation;
+ *    version 2.1 of the License.
+ *
+ *    This library is distributed in the hope that it will be useful,
+ *    but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ *    Lesser General Public License for more details.
+ */
+package org.geotools.appschema.filter;
+
+import static org.junit.Assert.assertNotNull;
+import static org.junit.Assert.assertTrue;
+
+import java.io.IOException;
+import java.io.Serializable;
+import java.net.URL;
+import java.util.HashMap;
+import java.util.Map;
+import org.geotools.data.DataAccess;
+import org.geotools.data.DataAccessFinder;
+import org.geotools.data.FeatureSource;
+import org.geotools.data.complex.feature.type.Types;
+import org.geotools.feature.FeatureCollection;
+import org.geotools.feature.FeatureIterator;
+import org.geotools.filter.FilterFactoryImpl;
+import org.geotools.test.AppSchemaTestSupport;
+import org.junit.BeforeClass;
+import org.junit.Test;
+import org.opengis.feature.Feature;
+import org.opengis.feature.Property;
+import org.opengis.feature.type.FeatureType;
+import org.opengis.feature.type.Name;
+import org.opengis.filter.Filter;
+import org.opengis.filter.expression.PropertyName;
+import org.xml.sax.helpers.NamespaceSupport;
+
+public class ArithmeticFilterTest extends AppSchemaTestSupport {
+
+    private static FilterFactoryImpl ff;
+
+    private static DataAccess<FeatureType, Feature> dataAccess;
+
+    private static FeatureSource<FeatureType, Feature> fSource;
+
+    @BeforeClass
+    public static void onetimeSetUp() throws Exception {
+
+        final String GSML_URI = "urn:cgi:xmlns:CGI:GeoSciML:2.0";
+        /** Set up filter factory */
+        NamespaceSupport namespaces = new NamespaceSupport();
+        namespaces.declarePrefix("gsml", GSML_URI);
+        namespaces.declarePrefix("gml", "http://www.opengis.net/gml");
+        ff = new FilterFactoryImplNamespaceAware(namespaces);
+
+        /** Load data access */
+        final Name FEATURE_TYPE = Types.typeName(GSML_URI, "MappedFeature");
+        final String schemaBase = "/test-data/";
+        Map<String, Serializable> dsParams = new HashMap<String, Serializable>();
+        dsParams.put("dbtype", "app-schema");
+        URL url = BBoxTest.class.getResource(schemaBase + "MappedFeatureAsOccurrence.xml");
+        assertNotNull(url);
+        dsParams.put("url", url.toExternalForm());
+        dataAccess = DataAccessFinder.getDataStore(dsParams);
+
+        fSource = (FeatureSource<FeatureType, Feature>) dataAccess.getFeatureSource(FEATURE_TYPE);
+    }
+
+    @Test
+    public void testWithArithmeticOperator() throws IOException {
+
+        // evaluating first feature with Location accuracy value = 200
+
+        // 200 x 2 = 400
+        Filter arthimeticMultiplyFilter =
```

Comment:
arthimetic ;-) (see other variables too)

Reply:
will fix spelling typos

Reply:
Spelling errors still seem to be here?

Reply:
missed those, will fix